### ISSUE TITLE
Bring back (or just allow) using old HEX format for items

### ISSUE THREAD

**Issue by konfix** [PluginBugs/Issues-ItemsAdder#3604]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

@konfa

### Describe the solution you'd like

I'd like to be able to continue using the old hex format %#123456% instead of the new <#123456>.
I have over a 1000 items that I have to manually change (I can't replace it since % needs to be < on the left and > on right side)

Plus, the new format works in both display_name and lore section, &#123456 only works in lores and not the display_name, and you can't mix formats (for example **<#123456>&l** -> it needs to be `<#123456><bold>` or `&#123456&l`) - a lot of symbols have &f before them and I have to think where to use <#ffffff> and where not, since you can't mix formats in the same string.

It's really time consuming, can you just bring back the support for old format alongside with the new one? It would save old users a LOT of time.

### Is your feature request related to a problem?

Yes, as explained above

### Describe alternatives you've considered

/

**Comment by Andre601** [comment]
From my personal perspective is the switch a good change, given that it utilizes MiniMessage syntax which is a lot more clear to see and understand, not to mention parse. Using `&` for start and end is a lot more complex, given you have to check on whether the `&` is the start or end of a color.

Regarding work: While re-adding this would reduce the work on your end would it add extra work on LoneDev's end, as he would need to re-implement this system. Even if it was just adding a parser would it remain as extra workload he now has to do. Always keep those things in mind.

Either way, we have to wate for LoneDev's response on this.

**Comment by konfix** [comment]
> From my personal perspective is the switch a good change, given that it utilizes MiniMessage syntax which is a lot more clear to see and understand, not to mention parse. Using `&` for start and end is a lot more complex, given you have to check on whether the `&` is the start or end of a color.
> 
> Regarding work: While re-adding this would reduce the work on your end would it add extra work on LoneDev's end, as he would need to re-implement this system. Even if it was just adding a parser would it remain as extra workload he now has to do. Always keep those things in mind.
> 
> Either way, we have to wate for LoneDev's response on this.

I understand and I know that this would probably get closed as it's technically considered a "misconfiguration" at this point, but it's a very big change that happened suddenly. I personally almost always use MiniMessage format when configuring, but some of my IA configs are 2 or more years old and they have the %% format.

If LoneDev chooses not to add the option back I completely understand and respect that, I'll do it the manual way, but the best option for users would be to support both, I've seen dozens of IA users that are annoyed with this change.

**Comment by pmgalileo** [comment]
I agree, I've been using IA for almost a year now, and after this update, I'm having a lot of trouble adapting all the items (more than 1600). I'm also of the opinion that the old format should be brought back

**Comment by KermX** [comment]
A config option to use old hex color codes may be best of both worlds? 
I'm fairly certain that using both next to each other is in no way worth the trouble. However, it would be incredibly beneficial to be able to use old format for those of us who have literally hundreds of items in the old format.

**Comment by LoneDev6** [comment]
The old format will be reintroduced.
Note that you can't mix minimessage, placeholder (%%) and legacy (&) color codes.

**Comment by LoneDev6** [comment]
I guess that 4 months passed and this feature is useless to be implemented at this point due to the delay.

**Comment by LoneDev6** [comment]
Will be implemented as some old servers still use it and they recently updated, but the old colors placeholders are not working anymore.
